.github/actions/spelling/allow/occ.txt

```diff
@@ -9,6 +9,7 @@ CLA
 contentpart
 CVC
 datetimefield
+deserializers
 disqus
```

Comment:
Maybe a warning message would be good here, if all payment providers are disabled: 
![image](https://github.com/OrchardCMS/OrchardCore.Commerce/assets/80963259/14f1760f-49c4-4302-8832-df2b3ff1474c)